IntelliTect.TestTools.Console/ConsoleAssert.cs

```diff
@@ -125,7 +182,7 @@ public static void Expect<T>(string expected, Func<T> func, T expectedReturn)
         /// <param name="func">Method to be run</param>
         /// <param name="args">Args to pass to the function.</param>
         public static void Expect(string expected, Action<string[]> func, params string[] args) =>
-            Expect(expected, () => func(args));
+            Expect(expected, () => func(args), NormalizeOptions.NormalizeLineEndings);
```

Comment:
Why not just rely on the parameter default - which means `NormalizeOptions.StripAnsiEscapeCodes` would be se.